packages/desktop-gui/cypress/integration/app_spec.js

```diff
@@ -0,0 +1,76 @@
+describe('App', function () {
+  beforeEach(() => {
+    cy.visitIndex().then(function (win) {
+      this.win = win; // don't remove this semicolon 😅
+
+      ({ start: this.start, ipc: this.ipc } = this.win.App)
```

Comment:
the semicolon should go at the beginning of this line, not the previous.

anytime you start a line in JS with `(` or `[` you need a preceding semicolon because javascript is not white space sensitive. imagine if you removed all the whitespace, then this opening parenthesis would represent a function invocation.

the semicolon prevents that... it's the same scenario as `[1,2,3].forEach()` needing to be `;[1,2,3].forEach()`

Reply:
I mean, I ideally didn't even want to use this semicolon. @brian-mann Is there a better way to write this? I got a little confused by the decaffeinate of the destructuring assignment tbh. 

Used to be this:
```
cy.visitIndex().then (@win) ->
  { @start, @ipc } = @win.App
```

Reply:
I think it's probably better to split it into 2 lines and forego destructuring:

```javascript
this.start = win.App.start
this.ipc = win.App.ipc
```